### PR TITLE
Uses correct key path for root command

### LINKED ISSUE #497: No default Key for root command (if multiple Tessels are plugged)
When running `t2 root` the default key is not set.
> INFO Starting SSH Session on Tessel. Type 'exit' at the prompt to end.
Warning: Identity file undefined not accessible: No such file or directory.
Permission denied (publickey).

reproduce this by plugging at least two Tessels and run `t2 root`

[x]    default key is present at ~/.tessel/id_rsa
[x]    Both Tessels are provisioned !
[ ]    `t2 root` also **should default to the default key** if it is not set by --key (it doesn't when multiple Tessels are plugged)

### PR DESCRIPTION
**Pull request by johnnyman727**
Fixes #497.

### CODE REVIEW

File: lib/controller.js [hunk 1]
```diff
@@ -744,7 +744,7 @@ controller.root = function(opts) {
       // Spawn a new SSH process
       var child = cp.spawn('ssh', ['-i',
         // Use the provided key path
-        opts.key,
+        Tessel.LOCAL_AUTH_KEY,
         // Connect to the Tessel's IP Address
```
Comment (Student007) [inline lib/controller.js]: @rwaldron from my point of view this doesn't make sense (also if it works).
@johnnyman727 thanks for that suggestion but could you explain why this is a better solution ?

Comment (johnnyman727) [inline lib/controller.js]: @Student007 in my opinion, it's the best solution because it's the least amount of changed code.

Comment (rwaldron) [inline lib/controller.js]: Doesn't this change prevent the end user from specifying a key? Or am I misunderstood? Should it be: `opts.key || Tessel.LOCAL_AUTH_KEY,`?

Comment (johnnyman727) [inline lib/controller.js]: No, it does not because the new key path gets set within Tessel.get.

Comment (rwaldron) [inline lib/controller.js]: Ah—Indeed! I overlooked the `controller.get` wrapper.

**Comment by rwaldron** [comment]
After reading through all of the associated issue comments and PRs, I'm going to land this because it's the simplest and likely easiest to further iterate on. Thanks to everyone that participated!!

**Pull request merged**